### DISCUSSION TITLE
Help Newbie

### DISCUSSION THREAD

**Discussion by Goldhdtv** [community/community#139909] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I need a developer to help me with JF/IPTV from the ground up

**Comment by ebndev** [comment]
Hi @Goldhdtv, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about your project / code snippets or a link to your repo. Giving a few more details might help someone give you a nudge in the right direction. 😄

**Reply by Goldhdtv** [reply]
I have my own iptv panel now and would like to maximize it as much as possible l. I've seen it done before because I was a reseller. Now that I have my own id like to give it a try. Or add it to JF if possible

**Comment by AbdulAhad365** [comment]
sure what is the project

**Reply by Goldhdtv** [reply]
I have a iptv panel and would like to maximize it as much as possible. I've seen it done by others because I've been a reseller before. Now I have my own and looking for someone who's familiar with it and know what I'm trying to do

**Reply by Goldhdtv** [reply]
Or add to Jfin if possible

**Comment by hamzamaimi** [comment]
I'm interested in this topic too. what's the matter?

**Reply by Goldhdtv** [reply]
I have my own iptv panel now and would like to maximize it as much as possible. I've seen it done before because I use to be a reseller. Now that I have my own I'm looking for a person to help me do that
